### PR TITLE
baseline video teaser test

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?

baseline for https://github.com/guardian/frontend/pull/13055
## What is the value of this and can you measure success?

Baselining
## Does this affect other platforms - Amp, Apps, etc?

No
## Screenshots

n/a
## Request for comment

@jamesgorrie

### CODE REVIEW

File: common/app/views/fragments/containers/facia_cards/videoContainer.scala.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h2 class="video-title fc-container__header__title">
                             autoPlay = false,
                             showControlsAtStart = false
                         )) { player =>
-                            <div class="fc-item__media-wrapper u-faux-block-link__promote media__container--hidden js-video-player">
+                            <div class="fc-item__media-wrapper u-faux-block-link__promote media__container--hidden js-video-player video-playlist__item__player">
```
Comment (jamesgorrie) [inline common/app/views/fragments/containers/facia_cards/videoContainer.scala.html]: Is this needed for the test?

File: common/app/conf/switches/ABTestSwitches.scala [hunk 2]
```diff
@@ -149,4 +149,12 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  val ABVideoTeaser = Switch(
+    SwitchGroup.ABTests,
+    "ab-video-teaser",
+    "Show video teaser",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 6, 1),
```
Comment (jamesgorrie) [inline common/app/conf/switches/ABTestSwitches.scala]: Is 1 day enough?

**Comment by jamesgorrie** [comment]
Bar the question about the added class 👍

File: static/src/javascripts/bootstraps/enhanced/media/main.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ define([
     'common/modules/commercial/build-page-targeting',
     'common/modules/commercial/commercial-features',
     'common/modules/component',
+    'common/modules/experiments/ab',
```
Comment (gtrufitt) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: What's the reason for defining this here and not using it?

File: common/app/views/fragments/containers/facia_cards/videoContainer.scala.html (hunk 1 above)
Comment (akash1810) [inline common/app/views/fragments/containers/facia_cards/videoContainer.scala.html]: nope - I branched off the other PR and squashed the commits... missed this change!

File: static/src/javascripts/bootstraps/enhanced/media/main.js (hunk 3 above)
Comment (akash1810) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: there isn't one! I branched off the other PR and squashed the commits... missed this change!

Comment (gtrufitt) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: :D 👍

**Pull request merged**